### PR TITLE
Refactor celery app to include taskflow queue

### PR DESCRIPTION
**Pull request by kotfic**
Note:  This will require changes to /etc/default/celeryd - in particular
the values:
```
CELERYD_NODES = "cumulus"
....
CELERYD_OPTS=""
....
```
This change will have to be made in HPCCloud-deploy once these changes
make thier way into master. ([here](https://github.com/Kitware/HPCCloud-deploy/blob/master/ansible/roles/celery/files/celeryd#L7) & [here](https://github.com/Kitware/HPCCloud-deploy/blob/master/ansible/roles/celery/files/celeryd#L22))

### CODE REVIEW

File: cumulus/celery/__init__.py [hunk 1]
```diff
@@ -19,6 +19,7 @@
 
 from __future__ import absolute_import
 from celery import Celery
+from kombu import Queue
```
Comment (cjh1) [inline cumulus/celery/__init__.py]: Should we be referencing this directly?

File: cumulus/mytaskflows.py [hunk 2]
```diff
@@ -93,7 +92,7 @@ class ChordTaskFlow(taskflow.TaskFlow):
     This taskflow has a "fanout" an also a chord, at task4
     """
     def start(self):
-        task1.delay(self)
+        task4.delay(self)
```
Comment (cjh1) [inline cumulus/mytaskflows.py]: Did you mean to commit this?

**Comment by cjh1** [comment]
LGTM, after you merge I will update your static analysis issues ...

File: cumulus/mytaskflows.py (hunk 2 above)
Comment (kotfic) [inline cumulus/mytaskflows.py]: This should be task1,  i just switched to task4 to jump right the chord for debugging.  FIxed in latest commit

**Pull request merged**